projects/robots/softbank/nao/controllers/nao_demo/nao_demo.c

```diff
@@ -321,8 +321,8 @@ static void print_help() {
   printf("----------nao_demo----------\n");
   printf("Select the robot and use the keyboard to control it:\n");
   printf("(The 3D window need to be focused)\n");
-  printf("[Up][Down]: move one step forward/backwards\n");
-  printf("[<-][->]: side step left/right\n");
+  printf("[Up][Down]: move few steps forward/backwards\n");
+  printf("[<-][->]: few side steps left/right\n");
```

Comment:
Here I would add a verb to be more consistent, e.g., "make a few side steps left/right"